content/en/community/hackathons/sessions/advanced-app-porting/content/Overview_Adding-New-Sections-to-an-ELF.md

```diff
@@ -66,55 +66,55 @@ Next, let's analyze the method by which we can go through this section to find t
 We must first import the endpoints of the section.
 It can be done as follows:
 
-```
+```C
 extern const struct my_structure my_section_start;
 extern const struct my_structure my_section_end;
 ```
 
 Using the endpoints we can write the macro for iterating through the section:
 
-```
+```C
 #define for_each_entry(iter)                                    \
         for (iter = &my_section_start;                          \
                 iter < &my_section_end;                         \
                 iter++)
 
 ```
 
-{{% alert title="Note" %}}
+{{< alert theme="info" >}}
 If you're not familiar with macros, you may check what they expand to with the GCC's preprocessor.
 Remove all the included headers and run `gcc -E main.c`.
-{{% /alert %}}
+{{</ alert >}}
 
 Let's configure the program.
 Use the `make menuconfig` command to set the KVM platform as in the following image.
 
-![platform_configuration](./images/platform_configuration.png)
+![platform_configuration](/community/hackathons/sessions/advanced-app-porting/images/platform_configuration.png)
 
 Save the configuration, exit the menuconfig tab and run `make`.
 Now, let's run it.
 You can use the following command:
 
-```
+```Bash
 $ qemu-guest -k build/01-extrald-app_kvm-x86_64
 ```
 
 The program's output should be the following:
 
-![01-extrald-app-output](./images/01-extrald-app-output.png)
+![01-extrald-app-output](/community/hackathons/sessions/advanced-app-porting/images/01-extrald-app-output.png)
```

Comment:
![image](https://user-images.githubusercontent.com/46280801/189075319-f844c1cb-d97d-448a-9422-9d74210518d9.png)

There's a typo in the image :pensive: 

I think we can skip this for now